### PR TITLE
update flower

### PR DESCRIPTION
**Pull request by hjmat**
- updates the version of flower, however the latest version forces you to use oauth (so by sticking with `0.7.3` we can still check on the version of celery we're using while retaining the existing authentication behavior).
- limits the memory usage
- compatible with a non deprecated heroku platform version

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-flower
\ No newline at end of file
+# sudo pip install -r requirements.txt
+flower==0.7.3
```
Comment (johnclarkemills) [inline requirements.txt:2]: I thought they were at 0.9 or something?

Comment (hjmat) [reply requirements.txt:2]: @johnclarkemills The version I tested with initially (0.9.2) hits this error `https://github.com/mher/flower/issues/343` which the author refuses to fix, unless you use oauth.

Comment (hjmat) [reply requirements.txt:2]: 0.7.3 is the latest version that does not suffer from that issue.

**Comment by johnclarkemills** [comment]
lgtm

**Comment by johnclarkemills** [comment]
@hjmat go ahead and merge this if the new app is all working.  also, please update the wiki with this new flower link

**Pull request merged**